js/components/geolocation.js

```diff
@@ -0,0 +1,186 @@
+Fliplet.FormBuilder.field('geolocation', {
+  name: 'Geolocation',
+  category: 'Advanced',
+  props: {
+    value: {
+      type: String
+    },
+    description: {
+      type: String
+    },
+    autofill: {
+      type: Boolean,
+      default: false
+    },
+    preciseLocationRequired: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data: function() {
+    return {
+      firstTimeSaved: false,
+      isLoading: false,
+      showFeedback: false,
+      accuracy: null,
+      timeOut: null
+    };
+  },
+  validations: function() {
+    var rules = {
+      value: {}
+    };
+
+    if (this.required && !this.readonly) {
+      rules.value.required = window.validators.required;
+    }
+
+    return rules;
+  },
+  mounted: function() {
+    if (this.autofill) {
+      this.getLocation();
+    }
+  },
+  created: function() {
+    Fliplet.FormBuilder.on('reset', this.onReset);
+    Fliplet.Hooks.on('beforeFormSubmit', this.onBeforeSubmit);
+  },
+  destroyed: function() {
+    Fliplet.FormBuilder.off('reset', this.onReset);
+    Fliplet.Hooks.off('beforeFormSubmit', this.onBeforeSubmit);
+  },
+  methods: {
+    setValue: function(result) {
+      return `${result.coords.latitude}, ${result.coords.longitude}`;
```

Comment:
Although we do allow spaces when we read data from a data source, the value format should not have a space when it's generated by the form. Please check this is the case everywhere.

```suggestion
      return `${result.coords.latitude},${result.coords.longitude}`;
```